### PR TITLE
Show carrier that applies to a claim on a claim

### PR DESCRIPTION
**Pull request by vonElfvin**
# Jira Issue: []

## What?
- Show the carrier on the claim

## Why?
- To ensure the receival of carrier works

## Optional screenshots
image

## Optional checklist
- [x] Updated `src/changelog.ts`
- [ ] Codescouted
- [ ] Unit tests written
- [x] Tested locally

### CODE REVIEW

File: schema.graphql [hunk 1]
```diff
@@ -937,25 +938,31 @@ enum ClaimSource {
...
+  state: ClaimState!
   type: ClaimType
   reserves: MonetaryAmount
-  registrationDate: Instant
-  notes: [ClaimNote]
-  transcriptions: [ClaimTranscription]
-  payments: [ClaimPayment]
-  events: [ClaimEvent]
+  registrationDate: Instant!
+  notes: [ClaimNote!]!
```
Comment (meletios) [inline schema.graphql:948]: Do we always have notes? not sure if this is true when the claim is just created for example.

**Review by meletios — COMMENTED** [review]
Looks good, one question regarding the schema

File: schema.graphql (hunk 1 above)
Comment (vonElfvin) [reply schema.graphql:948]: Yea it should always be at least an empty list

**Review by palmenhq — DISMISSED** [review]
Beautiful imo! Also nice with the schema improvements. However what about PR description checkboxes? :D Changelog, tested etc?

**Review by palmenhq — APPROVED** [review]

**Pull request merged**